flexget/plugins/output/json.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import os
+import logging
+
+from flexget import plugin
+from flexget.utils import json
+from flexget.event import event
+
+PLUGIN_NAME = 'make_json'
+
+log = logging.getLogger(PLUGIN_NAME)
+
+
+class OutputJson(object):
+    schema = {
+        'type': 'object',
+        'properties': {'file': {'type': 'string'}},
+        'required': ['file'],
+        'additionalProperties': False,
+    }
+
+    def on_task_output(self, task, config):
+
+        output = os.path.expanduser(config['file'])
+        # Output to config directory if absolute path has not been specified
+        if not os.path.isabs(output):
+            output = os.path.join(task.manager.config_base, output)
+
+        entries = {}
+        for entry in task.accepted:
+            entry_dict = dict(entry)
+            entries[entry_dict['title']] = entry_dict
```

Comment:
Should the plugin be configurable what the format is. What if someone wants to use url or some other field instead? I would have expected the output to be the most straightforward list of resources, ie.

```
[
  {'title': 'asdf1', 'url': 'foobar1'},
  {'title': 'asdf2', 'url': 'foobar2'},
  {'title': 'asdf3', 'url': 'foobar3'}
]
```